src/Microsoft.Kusto.ServiceLayer/DataSource/ReliableDataSourceConnection.cs

```diff
@@ -220,15 +186,6 @@ public void Close()
         {
         }
 
-        /// <summary>
-        /// Gets the time to wait while trying to establish a connection before terminating
-        /// the attempt and generating an error.
-        /// </summary>
-        public int ConnectionTimeout
```

Comment:
Same thing here. Check the "Advanced Properties" in the connection dialog. Remove the ones which are not needed both here and in the azuredatastudio repo.